src/BuildIt.Forms/BuildIt.Forms.Controls/Platforms/Android/CameraPreviewControlRenderer.cs

```diff
@@ -1020,9 +1109,24 @@ private void SetUpCameraOutputs(int width, int height)
             }
         }
 
+        private void ApplyAspect2()
```

Comment:
Perhaps "ApplyAspectForCamera2" or something to indicate what the "2" represents